website/docs/r/cis_waf_package.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_waf_package"
+sidebar_current: "docs-ibm-resource-cis-waf-package"
+description: |-
+  Provides a IBM CIS WAF Package resource.
```

Comment:
UI names it as OWASP rule set.. Can this be termed as alternate name in the docs..

Reply:
I followed CLI document.

Reply:
`OWASP rule set` can this also be mentioned as alternate name in docs

Reply:
Please specify the statement here which you propose. I will update the same in the doc.